crates/typos-cli/src/bin/typos-cli/report.rs

```diff
@@ -348,7 +481,7 @@ mod tests {
     fn test_calculate_visible_column_width_zwj_sequences() {
         let zwj_sequences = [
             "😵‍💫", /* U+1F635 U+200D U+1F4AB */
-            "👁️‍🗨️",   /* U+1F441 U+FE0F U+200D U+1F5E8 U+FE0F */
+            "👁️‍🗨️", /* U+1F441 U+FE0F U+200D U+1F5E8 U+FE0F */
```

Comment:
There are a lot of these small formatting changes.  At least move these to a separate commit

Reply:
the default formmater did this, any rules i can follow？

Reply:
Why is this marked as resolved when it isn't resolved?

Reply:
I just restored the original format. But apparently this causes the format check to fail in CI

Reply:
https://github.com/crate-ci/typos/pull/1048

Reply:
So let's leave it as it is.

Reply:
Ok